app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -39,7 +39,6 @@ import net.skyscanner.backpack.demo.stories.IconType
 import net.skyscanner.backpack.demo.stories.IconsStory
 import net.skyscanner.backpack.demo.stories.InteractiveStarRatingStory
 import net.skyscanner.backpack.demo.stories.LabeledCalendarStory
-import net.skyscanner.backpack.demo.stories.LoadingButtonStory
```

Comment:
We need to have a story when loading property changes in runtime

Reply:
TIL!

Reply:
good catch - this actually uncovered a bug - fixed that and also added a test for it